woocommerce/woocommerce
# Code refactor: Move marketing components into coupons directory

### All Submissions:


### Changes proposed in this Pull Request:

After we remove the classic Marketing page in PR https://github.com/woocommerce/woocommerce/pull/37430, some of the shared marketing components are only used by the Coupons page.

In this PR, we move those shared components into the coupons directory. The components are:

- `Card`
- `KnowledgeBase`
- `RecommendedExtensions`
- `Slider`

This PR is a code refactor / reorganization and should not change or break anything.

### How to test the changes in this Pull Request:




1. Go to Marketing page: `/wp-admin/admin.php?page=wc-admin&path=%2Fmarketing`. It should work the same as before.
2. Go to Coupons page: `/wp-admin/edit.php?post_type=shop_coupon`. It should work the same as before.

### Other information:


### FOR PR REVIEWER ONLY:
